packages/charts/src/components/ColumnChartWithTrend/ColumnWithTrendChart.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { render } from '@shared/tests';
+import * as React from 'react';
+import { complexDataSet } from '../../resources/DemoProps';
+import { ColumnWithTrendChart } from './ColumnWithTrendChart';
+
+describe('ColumnChart', () => {
+  test('Renders with data', () => {
+    const utils = render(
+      <ColumnWithTrendChart
+        dataset={complexDataSet}
+        style={{ width: '50%' }}
```

Comment:
not necessary for tests unless you want to test if the style is passed correctly. Also this will have no effect on the actual component since `JSDOM` has no mocked width or height, it is only reflected in the snapshot.